bin/trait_mapping/trait_mapping.py

```diff
@@ -0,0 +1,872 @@
+import argparse
```

Comment:
I'm not sure all this code should be in a single script. It is complex enough to require being made more modular, placing parts of it inside `eva_cttv_pipeline` and including unit tests.